.github/workflows/node.yml

```diff
@@ -72,19 +72,17 @@ jobs:
 
             - name: test hybrid node modules - commonjs
               run: |
-                cd hybrid-node-tests
-                cd commonjs-test
-                npm install 
+                npm install --package-lock-only
+                npm ci 
                 npm run build-and-test
-              working-directory: ./node
+              working-directory: ./node/hybrid-node-tests/commonjs-test
             
             - name: test hybrid node modules - ecma
               run: |
-                cd hybrid-node-tests
-                cd ecmascript-test
-                npm install
+                npm install --package-lock-only
+                npm ci
```

Comment:
I don't think you need to do both. 👀 

I'm just wondering "what's the best thing to do"?

The `npm ci` is usually sufficient on its own https://docs.npmjs.com/cli/v6/commands/npm-ci However, I don't know how it works for different OS, like Linux / macOS and Windows dependencies. Theoretically, it shouldn't be a problem.

Reply:
unfortunately npm ci fails since there no package-lock (broke the ci in the pervious push), so theres a need for to create a lock file to run npm ci.

Reply:
Oh! Ok, make sens ! Sorry :eyes: